packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -401,6 +419,17 @@ export class IncomingRequestsController extends ConsumptionBaseController {
         }
     }
 
+    private async checkRelationshipStatus(peer: CoreAddress): Promise<ValidationResult> {
+        const relationship = await this.relationshipResolver.getRelationshipToIdentity(peer);
+        // no relationship is for a template request on new relationship
+        if (relationship && relationship.status !== RelationshipStatus.Active) {
```

Comment:
a decent amount of the transport library does it the other way - I guess we can try to agree on something?!

Reply:
maybe we should go away from that ugly typeof thing. Please leave these things as is in this PR.

Reply:
I remember tons of these comments in my PRs and you arguing in favor of it. 😅 I don't feel strongly about this, though. But maybe we can discuss this shortly in the next sync, so that we have a guide line for the following PRs.

Reply:
and I definitely had reasons about that. But opinions change.

> But maybe we can discuss this shortly in the next sync

Yes, please!